secant/Dependencies/SDKSynchronizer/SDKSynchronizerLive.swift

```diff
@@ -17,6 +17,9 @@ enum SDKSynchronizerDependency: DependencyKey {
 class LiveSDKSynchronizerClient: SDKSynchronizerClient {
     private var cancellables: [AnyCancellable] = []
     private(set) var synchronizer: SDKSynchronizer?
+    /// TODO: Since 0.17.0-beta SDKSynchronizer has `lastState` property which does exactly the same as `stateChanged`. Problem is that we have
+    /// synchronizer as optional. And now it would be complicated to handle the situation when `lastState` isn't always available. Let's handle this
+    /// in future.
```

Comment:
Can you add an issue to track this ?

Reply:
https://github.com/zcash/secant-ios-wallet/issues/497